packages/frontend-2/components/server-management/DeleteProjectDialog.vue

```diff
@@ -0,0 +1,142 @@
+<template>
+  <LayoutDialog
+    v-model:open="isOpen"
+    max-width="sm"
+    :title="title"
+    :buttons="dialogButtons"
+  >
+    <div class="flex flex-col gap-6">
+      <p>
+        Are you sure you want to
+        <strong>permanently delete</strong>
+        the selected project?
+      </p>
+      <div v-if="project">
+        <strong>{{ project.name }}</strong>
+        <p>
+          {{ project.models.totalCount }} models,
+          {{ project.versions.totalCount }} versions,
+        </p>
+      </div>
+      <p>
+        This action
+        <strong>cannot</strong>
+        be undone.
+      </p>
+    </div>
+  </LayoutDialog>
+</template>
+
+<script setup lang="ts">
+import { LayoutDialog } from '@speckle/ui-components'
+import { ProjectItem } from '~~/lib/server-management/helpers/types'
+import { useMutation } from '@vue/apollo-composable'
+import {
+  convertThrowIntoFetchResult,
+  getCacheId,
+  getFirstErrorMessage,
+  updateCacheByFilter
+} from '~~/lib/common/helpers/graphql'
+import { useGlobalToast, ToastNotificationType } from '~~/lib/common/composables/toast'
+import { adminDeleteProject } from '~~/lib/server-management/graphql/mutations'
+import { getProjects } from '~~/lib/server-management/graphql/queries'
+import { Exact, InputMaybe } from '~~/lib/common/generated/gql/graphql'
+
+const props = defineProps<{
+  title: string
+  open: boolean
+  project: ProjectItem | null
+  resultVariables:
+    | Exact<{
+        limit: number
+        cursor?: InputMaybe<string> | undefined
+        query?: InputMaybe<string> | undefined
+      }>
+    | undefined
+}>()
+
+const emit = defineEmits<{
+  (e: 'update:open', val: boolean): void
+}>()
+
+const { triggerNotification } = useGlobalToast()
+const { mutate: adminDeleteMutation } = useMutation(adminDeleteProject)
+
+const isOpen = computed({
+  get: () => props.open,
+  set: (newVal) => emit('update:open', newVal)
+})
+
+const deleteConfirmed = async () => {
+  const projectId = props.project?.id
+  if (!projectId) {
+    return
+  }
+
+  const result = await adminDeleteMutation(
+    {
+      ids: [projectId]
+    },
+    {
+      update: (cache, { data }) => {
+        if (data?.streamsDelete) {
+          // Remove invite from cache
+          cache.evict({
+            id: getCacheId('AdminUserListItem', projectId)
```

Comment:
This couldn't have ever worked, because the list has items of type `Project` not `AdminUserListItem`. I've noticed this issue in multiple dialogs (e.g. Delete invites dialog, which I fixed). Please check that your cache updates work properly.